default/python/AI/AIDependencies.py

```diff
@@ -349,21 +331,13 @@ def not_affect_by_special(special: str, species: str) -> bool:
 NEST_DOMESTICATION_TECH = "SHP_DOMESTIC_MONSTER"
 
 LRN_ARTIF_MINDS_1 = "LRN_NASCENT_AI"
-LRN_ARTIF_MINDS_2 = "LRN_NASCENT_AI"
 LRN_ALGO_ELEGANCE = "LRN_ALGO_ELEGANCE"
 GRO_PLANET_ECOL = "GRO_PLANET_ECOL"
-GRO_SUBTER_HAB = "GRO_SUBTER_HAB"
-LRN_PHYS_BRAIN = "LRN_PHYS_BRAIN"
```

Comment:
This values are likely redundantly defined in the tech lists. I guess I'll sort it out when working on tech, for now they can go.